mender/templates/deployments-deploy.yaml

```diff
@@ -90,11 +92,6 @@ spec:
 {{- if and .Values.auditlogs.enabled .Values.global.enterprise }}
         - name: DEPLOYMENTS_ENABLE_AUDIT
           value: "true"
-{{- end }}
-{{- if or .Values.global.s3.AWS_SERVICE_ACCOUNT_NAME (not .Values.global.s3.AWS_ACCESS_KEY_ID) }}
-  # this is required because deprecated func is used https://github.com/mendersoftware/deployments/blob/554b194e1ff389e50200644b8034bb012ede197a/s3/filestorage.go#L114
-        - name: AWS_SDK_LOAD_CONFIG
-          value: "1"
```

Comment:
Are we sure we don't need AWS_SDK_LOAD_CONFIG anymore?

Reply:
According to @alfrunes, yes. Alf, please confirm

Reply:
Deployments service does not use this parameter, I'm not entirely sure if we have any other dependencies that are still using aws-sdk-go (v1).